frontend/src/views/website/website/config/php/version/index.vue

```diff
@@ -92,16 +88,23 @@ const submit = async () => {
             try {
                 await ChangePHPVersion(versionReq);
                 MsgSuccess(i18n.global.t('commons.msg.updateSuccess'));
+                getWebsiteDetail();
             } catch (error) {}
             loading.value = false;
         });
     } catch (error) {}
 };
 
+const getWebsiteDetail = async () => {
+    const res = await GetWebsite(props.id);
+    versionReq.runtimeID = res.data.runtimeID;
+    oldRuntimeID.value = res.data.runtimeID;
+};
+
 onMounted(() => {
-    versionReq.runtimeID = props.runtimeID;
     versionReq.websiteID = props.id;
-    oldRuntimeID.value = props.runtimeID;
+
+    getWebsiteDetail();
     getRuntimes();
 });
 </script>
```

Comment:
None found. The code provided has no obvious issues or irregularities but could be improved with more specific and clear documentation of its purpose and usage.

However, it's important to note that for the current knowledge cutoff date you specified (September 2021), there likely won't be any significant changes or enhancements made after then. As such, further optimizations might not be applicable based on what you have at your disposal right now.